README.md

```diff
@@ -1,9 +1,27 @@
 # Northcoders News API
 
-## Background
+## About
 
-This repository mimics a real world backend service (such as reddit) which should provide this information to the front end architecture. It uses PSQL and interacts with it using [node-postgres](https://node-postgres.com/).
+Welcome to ncnews. This repository mimics a real world backend service (such as reddit) which should provide this information to the front end architecture. It uses PSQL and interacts with it using [node-postgres](https://node-postgres.com/).
+
+Data is posted in `articles` which have `votes` and can have related `comments`. Articles can also be categorised by `topics`. Both comments and articles are linked to `users`.
```

Comment:
It might be easier (and more helpful to someone interested in how this is all setup) to somehow show your db schema.  Obviously in the 'real world' we wouldn't be posting all this information, but as a portfolio piece it can be helpful.  There are several ways to do this, a graphic could be pretty useful.

A starting point to look at: https://www.educative.io/blog/what-are-database-schemas-examples